src/bluesearch/database/topic.py

```diff
@@ -0,0 +1,228 @@
+# Blue Brain Search is a text mining toolbox focused on scientific use cases.
+#
+# Copyright (C) 2020  Blue Brain Project, EPFL.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+"""Utils for journal/articles topics."""
+import pathlib
+from typing import Dict, Iterable, List, Union
+from xml.etree.ElementTree import Element  # nosec
+
+import requests
+from defusedxml import ElementTree
+
+
+# Journal Topic
+def get_mesh_from_nlm_ta(nlm_ta: str) -> List[Dict[str, Union[str, List[str]]]]:
+    """Retrieve Medical Subject Heading from Journal's NLM Title Abbreviation.
+
+    Parameters
+    ----------
+    nlm_ta : str
+        NLM Title Abbreviation of Journal.
+
+    Returns
+    -------
+    mesh :
```

Comment:
Does it actually raise an error/warning? If yes, we should be able to catch this in the CI, right?

Reply:
The CI was passing with the previous implementation. 

Change done in 0fca7c3.

Reply:
> The CI was passing with the previous implementation.

I think that if you don't specify the return type, then `sphinx` doesn't error out, but if you look at the API docs produced as an output you will see that the return type was not successfully inferred.

Reply:
> Change done in 0fca7c3.

Awesome, thanks a lot @EmilieDel !

I am not resolving this conversation because of @jankrepl 's comment which I think has not been addressed yet.
I guess he's suggesting to add something like
```python
    Raises
    ------
    ElementTree.ParseError
        If more than one result ...
```
after then `Return` in the docstring.

Reply:
Done in 0c200b0.